### PR TITLE
Sub/feature/organization details page

### PR DESCRIPTION
**Pull request by tanmaythole**

### CODE REVIEW

File: src/Components/facetAdmin/OrgDetails/OrgDetailTab.jsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        <small>
+          <b>Organization Name:</b>
+        </small>
+        <div className="text-capitalize">{name}</div>
+      </Col>
+      <Col className="py-2">
+        <small>
+          <b>Description:</b>
+        </small>
+        <div>{description ?? "-"}</div>
```
Comment (ChinmaySonawane) [inline src/Components/facetAdmin/OrgDetails/OrgDetailTab.jsx]: Can we try || here?

File: src/Components/facetAdmin/OrgDetails/OrgMembersList.jsx [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+          {
+            name,
+            username,
+            email,
+            is_admin: isAdmin,
+            is_super_admin: isSuperAdmin,
+          },
+          index
+        ) => (
+          <tr key={index}>
```
Comment (ChinmaySonawane) [inline src/Components/facetAdmin/OrgDetails/OrgMembersList.jsx]: Can we write separate component for this

File: src/Components/facetAdmin/OrgDetails/OrgMembersList.jsx [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+            <td>{index + 1}</td>
+            <td>
+              <div>{name}</div>
+              <small className="text-muted">{username}</small>
+            </td>
+            <td>{email}</td>
+            <td>{convertRolesToStr(isSuperAdmin, isAdmin)}</td>
```
Comment (ChinmaySonawane) [inline src/Components/facetAdmin/OrgDetails/OrgMembersList.jsx]: need to use const for convertRolesToStr(isSuperAdmin, isAdmin)

File: src/Components/facetAdmin/OrgDetails/OrgDetailTab.jsx (hunk 1 above)
Comment (tanmaythole) [reply src/Components/facetAdmin/OrgDetails/OrgDetailTab.jsx]: 👍🏻

File: src/Components/facetAdmin/OrgDetails/OrgMembersList.jsx (hunk 2 above)
Comment (tanmaythole) [reply src/Components/facetAdmin/OrgDetails/OrgMembersList.jsx]: 👍🏻

File: src/Components/facetAdmin/OrgDetails/OrgMembersList.jsx (hunk 3 above)
Comment (tanmaythole) [reply src/Components/facetAdmin/OrgDetails/OrgMembersList.jsx]: 👍🏻

**Review by ChinmaySonawane — APPROVED** [review]

**Pull request merged**